m127_2 release (#4684)
firebase/firebase-android-sdk: firebase-crashlytics/CHANGELOG.md

# Unreleased

> * [fixed] Fixed NPE if firebase-common is at too low a version

Reviewer: May a suggest rewording this as:
```suggestion
* [fixed] Fixed NPE caused by an older version of `firebase-common`
```

or even be more explicit and say:
```suggestion
* [changed] Updated to depend on `firebase-common:20.3.0`.
```

Reviewer 2: updated to match rachel's spec
